app.json

```diff
@@ -3,10 +3,10 @@
   "repository": "https://github.com/alphagov/govuk-coronavirus-find-support",
   "env": {
     "GOVUK_APP_DOMAIN": {
-      "value": "www.gov.uk"
+      "value": "www.gov.wales"
```

Comment:
we don't use the www prefix, can this be changed to just gov.wales?

we also use a different domain depending on the language (either gov.wales or llyw.cymru) - so will probably have to address that too, but can do that in a follow-up if you'd prefer?

Reply:
Ah yes, I'll change that.

I'm struggling a bit to determine how these environment variables are actually used.  In the gems that we include, I can only find 2 uses of  `GOVUK_APP_DOMAIN`. (Content-security-policy and Google Analytics)

https://github.com/alphagov/govuk_app_config/blob/v2.1.2/lib/govuk_app_config/govuk_content_security_policy.rb#L13
https://github.com/alphagov/govuk_publishing_components/blob/v21.42.0/app/views/govuk_publishing_components/components/_admin_analytics.html.erb#L9

I can't find any relevant uses of `GOVUK_WEBSITE_ROOT`

Reply:
I think it came in to play when we were using S3. So maybe we can remove it now?